apps/server/src/routes/chat.ts

```
@@ -351,33 +372,86 @@ export class ZeroAgent extends AIChatAgent<typeof env> {
   }
 }
 
-export class ZeroMCP extends McpAgent<typeof env, {}, { cookie: string }> {
-  auth: SimpleAuth;
+export class ZeroMCP extends McpAgent<typeof env, {}, { userId: string }> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)